manager/integration/tests/test_ha.py

```diff
@@ -233,6 +233,42 @@ def ha_salvage_test(client, core_api, # NOQA
 
     cleanup_volume(client, volume)
 
+    # test auto salvage when revision counter enabled ordisabled.
+    # One for revision counter enabled case.
+    test_ha_salvage_autosalvage()
+
+    # One for revision counter disabled case.
+    test_ha_salvage_autosalvage()
+
+
+@pytest.mark.skip(reason="TODO")
+def test_ha_salvage_autosalvage():
```

Comment:
And you can rename the salvage related tests to `test_ha_auto_salvage_with_revision_counter`, `test_ha_auto_salvage_without_revision_counter`, `test_ha_manual_salvage_with_revision_counter`, and `test_ha_manual_salvage_without_revision_counter`.